Proofread administration (#2681)
centreon/centreon-documentation: versioned_docs/version-23.10/administration/database-partitioning.md

Partitioning uses XML files in the **/usr/share/centreon/config/partition.d/**
directory to create the necessary partitions.

Every day, a script launched by a cron creates missing tables

> or those created in advance:

Reviewer: ```suggestion
or creates new tables in advance:
```

directory to create the necessary partitions.

> Every day, a script launched by a cron realizes the creation of missing tables

Reviewer: ```suggestion
Every day, a script launched by a cron creates missing tables
```

## Migrating unpartitioned tables

The command line performs the following procedure:

- Rename existing table (‘xxx’ will be ‘xxx\_old’)
- Create an empty partitioned table
- Migrate data into the partitioned table (with ‘SELECT INSERT’ statement)

> You need to check a few things before:

Reviewer: ```suggestion
You need to check a few things before executing the command:
```

> - Enough space on the MariaDB Server (at least twice the size of the table.

Reviewer: ```suggestion
- Enough space on the MariaDB Server (at least twice the size of the dtata and indexes)
```

> Indexes and Data)

Reviewer: ```suggestion
```
